### ISSUE TITLE
Vector logo source

### ISSUE THREAD

**Issue by xperiandri** [elmish/Elmish.WPF#389]
Is there a vector logo source for us to change the text `Elmish WPF` to `Elmish Uno` and use it in Elmish.Uno package?

**Comment by TysonMN** [comment]
Yes.  The file [elmish-wpf-logo.afdesign](https://github.com/elmish/Elmish.WPF/blob/21c98b43700bfc5a1e3d71a755b461be6cddcdbb/logo/elmish-wpf-logo.afdesign) is an [Affinity Designer](https://affinity.serif.com/en-us/) source file.  This program is a competitor of Photoshop that is _significantly_ more affordable.  Normally it costs $50, but I see that it is currently on sale for $25 (both are one-time costs, not a subscription).
